external/js/admin.js

```diff
@@ -22,7 +22,7 @@ $(document).ready(function(){
 	}
 
 	function saveSites() {
-		var post = $('#external').serialize();
+		var post = $('#external form').serialize();
```

Comment:
This Xpath doesn't look good, at least with the template I see below. It should be either `form#external` (which doesn't make sense) or just `#external`. Notice that the `#external` id MUST be unique, otherwise we might be using the wrong element.